installation/language/de-DE/de-DE.ini

```diff
@@ -3,123 +3,92 @@
-INSTL_STEP_DATABASE_LABEL="Datenbank"
-INSTL_STEP_DEFAULTLANGUAGE_LABEL="Standard-Sprache auswählen"
-INSTL_STEP_FTP_LABEL="FTP"
-INSTL_STEP_LANGUAGES_LABEL="Sprachen installieren"
-INSTL_STEP_SITE_LABEL="Konfiguration"
-INSTL_STEP_SUMMARY_LABEL="Überblick"
+;Build incomplete error page
+;These will be processed by the JavaScript Build
+BUILD_INCOMPLETE_LANGUAGE="Deutsch"
+BUILD_INCOMPLETE_HEADER="Umgebungseinstellungen unvollständig"
+BUILD_INCOMPLETE_TEXT="Es sieht so aus, als ob Sie versuchen, Joomla! aus dem Git-Repository herraus zu installieren. Um dies zu tun, müssen Sie zuerst ein paar zusätzliche Schritte durchführen."
+BUILD_INCOMPLETE_URL_TEXT="Weitere Informationen"
+;End of build incomplete error page
 
-;Language view
+;Minimum PHP error page
+;These will be processed by the JavaScript Build
+MIN_PHP_ERROR_LANGUAGE="Deutsch"
+MIN_PHP_ERROR_HEADER="Sorry, deine PHP-Version wird nicht unterstützt."
+MIN_PHP_ERROR_TEXT="Ihr Host muss die PHP-Version {{phpversion}} oder neuer verwenden, um diese Version von Joomla auszuführen."
+MIN_PHP_ERROR_URL_TEXT="Hilf mir, dies zu lösen."
+;End of minimum PHP error page
+
+;Main Config
+INSTL_SELECT_INSTALL_LANG="Installationssprache auswählen"
 INSTL_SELECT_LANGUAGE_TITLE="Sprachauswahl"
 INSTL_WARNJAVASCRIPT="Warnung! JavaScript muss für eine reibungslose Installation von Joomla! aktiviert sein!"
 INSTL_WARNJSON="In der PHP-Umgebung muss JSON aktiviert sein, damit Joomla! installiert werden kann!"
+INSTL_SETUP_LOGIN_DATA="Zugangsdaten einrichten"
 
-;Preinstall view
-INSTL_PRECHECK_TITLE="Installationsprüfung"
-INSTL_PRECHECK_DESC="Sollte einer der Einträge vom Server nicht unterstützt werden, mit einem „<strong class=\"label label-important\">Nein</strong>“ gekennzeichnet, bitte die Einstellungen auf dem Server prüfen und anpassen.<br />Joomla! kann nicht installiert werden, wenn die unten aufgeführten Systemvoraussetzungen nicht erfüllt sind."
-INSTL_PRECHECK_RECOMMENDED_SETTINGS_TITLE="Empfohlene Einstellungen:"
-INSTL_PRECHECK_RECOMMENDED_SETTINGS_DESC="Diese Einstellungen werden für PHP empfohlen, um eine gute Kompatibilität mit Joomla! zu gewährleisten.<br />Jedoch kann Joomla! hier mit Einschränkungen in den Empfehlungen trotzdem funktionieren."
+;Precheck view
+INSTL_DATABASE_SUPPORT="Datenbankunterstützung:"
+INSTL_JSON_SUPPORT_AVAILABLE="JSON-Support"
+INSTL_MB_LANGUAGE_IS_DEFAULT="PHPs mbstring Spracheinstellung ist neutral"
+INSTL_MB_STRING_OVERLOAD_OFF="PHPs „mbstring function overload“ ist deaktiviert"
+INSTL_NOTICE_DATABASE_SUPPORT="Es wurden keine unterstützten Datenbanken gefunden."
+INSTL_NOTICE_JSON_SUPPORT_AVAILABLE="In der PHP-Umgebung muss JSON aktiviert sein, damit Joomla! installiert werden kann!"
+INSTL_NOTICE_MBLANG_NOTDEFAULT="PHPs mbstring Spracheinstellung ist nicht auf „neutral“ gesetzt. Dies kann lokal eingestellt werden, indem Sie <strong>php_value mbstring.language neutral</strong> in Ihre <code>.htaccess</code> Datei eingeben."
```

Comment:
INSTL_NOTICE_MBLANG_NOTDEFAULT="PHPs mbstring Spracheinstellung ist nicht auf „neutral“ gesetzt. Dies kann lokal eingestellt werden, indem eine neue Zeile mit <strong>php_value mbstring.language neutral</strong> in die <code>.htaccess</code> Datei eingefügt wird."